components/Search/SearchResults/index.js

```diff
@@ -0,0 +1,27 @@
+import { interviews } from "@/public/interviews";
+
+export default function SearchResults({ searchTerm }) {
+  const results = interviews.filter((interview) =>
+    interview.tags.some((tag) => tag.includes(searchTerm))
+  );
+
+  if (!searchTerm || !results.length) {
+    return <p>No results</p>;
+  }
+
+  return (
+    <ul>
+      {results.map((interview) => (
+        <li key={interview.id}>
+          <h3>{interview.title}</h3>
+          <p>{interview.text}</p>
+          <ul>
+            {interview.tags.map((tag) => (
+              <li key={tag}>{tag}</li>
+            ))}
+          </ul>
+        </li>
+      ))}
```

Comment:
I'm not sure if you need to offer the entire article as a search result. A link to the article page would probably be more than enough. If you want to include a preview of the article text, you might want to add some overflow rules, such as

```js
//...

<PreviewText>{interview.text}</PreviewText>

//...

const PreviewText = styled.p`
text-overflow: ellipsis;
white-space: nowrap;
overflow: hidden;
`
```
Maybe you'd also not want to include the raw markdown text?

Reply:
Looks better. I tried the preview for now and maybe make a new US and focus on refining the search. Will have to sketch out something during the weekend.